packages/playground/src/dashboard/components/public_ips_table.vue

```diff
@@ -163,19 +185,24 @@ export default {
       }
       loadingIps.value = false;
     }
-    async function removeFarmIp(options: RemoveFarmIPModel, index: number) {
+    async function removeFarmIps() {
       try {
+        showDialogue.value = false;
         isRemoving.value = true;
-        await gridStore.grid.farms.removeFarmIp({ ip: options.ip, farmId: options.farmId });
-        createCustomToast("IP is deleted successfully!", ToastType.success);
-        publicIps.value.splice(index, 1);
+        for (const item of itemsToDelete.value) {
+          await gridStore.grid.farms.removeFarmIp({
+            ip: item.ip,
+            farmId: props.farmId,
+          });
+        }
```

Comment:
this makes things take time, we can add an method to grid client that uses batchall to delete all of them in one request